heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -110,6 +110,19 @@ local function get_os_version()
 end
 
 
+local function get_profile_creation()
+    local json = read_message("Fields[environment.profile]")
+    local ok, profile = pcall(cjson.decode, json)
+    if not ok then return 0 end
+
+    local days = profile.creationDate
+    if type(days) == "number" and days > 0 then
+        return days * SEC_IN_DAY * 1e9
+    end
+    return 0
```

Comment:
I think we should default to "today" as the creation date if it is missing. But that's just my opinion, we should confirm with the metrics team.